app/main.py

```diff
@@ -1,8 +1,23 @@
 class Person:
-    # write your code here
-    pass
+    people: dict = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = [Person(human["name"], human["age"]) for human in people]
+
+    def find_human(name: str) -> Person:
+        return next((human for human in person_list if human.name == name), None)
+
+    for human in people:
+        if "wife" in human and human["wife"] is not None:
+            husband = find_human(human["name"])
+            wife = find_human(human["wife"])
+            husband.wife = wife
+            wife.husband = husband
```

Comment:
Similarly, ensure that `husband` is not `None` before assigning `wife.husband` to avoid potential errors.